### DISCUSSION TITLE
docs/ninjutsu/medium-ranking-ninja/like-touching-fishing/

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#16] [General]
# docs/ninjutsu/medium-ranking-ninja/like-touching-fishing/

咋一看平平无奇，再一瞄毫无新意，但是隐藏在背后的是！！！

https://996.ninja/docs/ninjutsu/medium-ranking-ninja/like-touching-fishing/

**Comment by Ele-Cat** [comment]
猿粪啊，逛摸鱼站逛到了自己的摸鱼站 :see_no_evil:

**Reply by MingGH** [reply]
哈哈哈哈哈😹，一定是特别的猿粪
